<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaViewForm</class>
 <widget class="QWidget" name="MediaViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_pbEdit">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="../main.qrc">
         <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbPurge">
       <property name="text">
        <string>Purge</string>
       </property>
       <property name="icon">
        <iconset resource="../main.qrc">
         <normaloff>:/images/purge.png</normaloff>:/images/purge.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../main.qrc">
         <normaloff>:/images/purge.png</normaloff>:/images/purge.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbPrune">
       <property name="text">
        <string>Prune</string>
       </property>
       <property name="icon">
        <iconset resource="../main.qrc">
         <normaloff>:/images/edit-cut.png</normaloff>:/images/edit-cut.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filter</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Media Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="m_cbMediaType"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="m_cbStatus"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Limit:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="m_sbLimit">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="m_textName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="8" colspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Pool:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="m_cbPool"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="m_cbLocation"/>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QCheckBox" name="m_chkExpired">
        <property name="text">
         <string>Expired</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7" colspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="9">
       <widget class="QPushButton" name="m_pbApply">
        <property name="text">
         <string>Apply</string>
        </property>
        <property name="icon">
         <iconset resource="../main.qrc">
          <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="m_tableMedia">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string> Volume Name </string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Online</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Slot</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vol Bytes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vol Usage</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Vol Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pool</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Media Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last Written</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>When expire?</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections/>
</ui>
